ui/components/app/snaps/snap-ui-renderer/components/input.ts

```diff
@@ -19,13 +19,11 @@ export const constructInputProps = (props: InputElement['props']) => {
 
       return {
         type,
+        step,
+        min,
+        max,
         textFieldProps: {
-          type,
-          inputProps: {
-            step: step?.toString(),
-            min: min?.toString(),
-            max: max?.toString(),
-          },
+          type: 'text',
```

Comment:
Is it intentional to use `text` here? Wouldn't we want to use `number` inputs in case the browser supports it?